### ISSUE TITLE
[mio-tflirte260] ARM32 build fails

### ISSUE THREAD

**Issue by seanshpark** [Samsung/ONE#8622]
ARM32 cross build fails with
```
compiler/mio-tflite260/gen/mio/tflite/schema_generated.h:594:20: error: ‘IsOutRange’ is not a member of ‘flatbuffers’
   if (flatbuffers::IsOutRange(e, TensorType_FLOAT32, TensorType_UINT32)) return "";
                    ^~~~~~~~~~
```

and lots of `not membofer of tflite::...`

**Comment by seanshpark** [comment]
Seems related with #8397

**Comment by seanshpark** [comment]
Maybe #8492 will fix this :)

**Comment by seanshpark** [comment]
this issue is resolved by using one FB 2.0
